index.js

```diff
@@ -32,9 +33,7 @@ module.exports = {
   },
 
   shouldIncludeFiles: function() {
-    var config = this.app.project.config()['ember-cli-mirage'];
-
-    return config.force || (this.app.env !== 'production')
+    return this.addonConfig.force || !this.addonConfig.usingProxy ? this.addonConfig.isEnabledEnvironment : true;
```

Comment:
during `ember serve` is there two builds for each environment (dev, test)? if there isn't this may not work